package.json

```diff
@@ -5317,19 +5380,19 @@
 				"icon": "$(rocket)"
 			},
 			{
-				"command": "gitlens.focus.split",
+				"command": "gitlens.launchpad.split",
 				"title": "Split Launchpad in Editor",
 				"category": "GitLens",
 				"icon": "$(split-horizontal)"
 			},
 			{
-				"command": "gitlens.focus.experimental.updateIndicators",
-				"title": "Update Launchpad Indicators",
+				"command": "gitlens.launchpad.indicator.update",
```

Comment:
Is this a refresh? Maybe call it `Refresh Launchpad Indicator`?

Reply:
It's not really a refresh - it's essentially a command to update one of the indicator settings. Maybe I can change it to `updateSettings`

Reply:
Ehh now that I think about it, it also does things like 'connect to GitHub' etc. so maybe `update` is fine for now until we can think of a better name for what this is doing:

```
			registerCommand('gitlens.launchpad.indicator.update', async (action: string) => {
				switch (action) {
					case 'mute':
						void configuration.updateEffective('launchpad.indicator.polling.enabled', false);
						break;
					case 'unmute':
						void configuration.updateEffective('launchpad.indicator.polling.enabled', true);
						break;
					case 'hide': {
						const action = await window.showInformationMessage(
							'Would you like to hide the Launchpad status bar icon? You can re-enable it at any time using the "GitLens: Toggle Launchpad Status Bar Icon" command.',
							{ modal: true },
							'Hide',
						);
						if (action === 'Hide') {
							void configuration.updateEffective('launchpad.indicator.enabled', false);
						}
						break;
					}
					case 'connectGitHub': {
						const github = await this.container.integrations?.get(HostingIntegrationId.GitHub);
						if (github == null) break;
						if (!(github.maybeConnected ?? (await github.isConnected()))) {
							void github.connect();
						}
						break;
					}
					default:
						break;
				}
			}
```